UKHomeOffice/cerberus-service
# fixes bug where buy task details page does not render when booking date is null

## Description
This PR is a bug fix for certain tasks whereby task details page was not being rendered if booking date is null.

## To Test
- Pull & run against dev
- Load up this task [http://localhost:8080/tasks/AUTOTEST-02-03-2022-RORO-Tourist-OCCUPANT-COUNTS_440675:CMID=TEST](url) or [http://localhost:8080/tasks/COKAFOR-ACCOMPANIED-2022-03-1-1](url)
- `OUTCOME:` The page should load successfully.
## Developer Checklist

\* Required
